src/pages/namespace/Settings/Secrets/index.tsx

```diff
@@ -46,6 +48,12 @@ const SecretsList: FC = () => {
 
   const { mutate: deleteSecretMutation } = useDeleteSecret({
     onSuccess: () => {
+      /**
+       * When the user has uninitialized secrets, this will be shown behind
+       * the notification bell. The user might just have deleted that secret
+       * and we need to update the notification bell.
+       */
```

Comment:
This one comes from me, so I might answer that. I wanted to explain why exactly we refetch notifications here, because reading the code alone will not give you any hint why that is necessary. Like explained in my comment, I don't like that we have to do refetching here, but as long as we don't have a better solution, I think it is a good idea to exactly explain these notifications side effect.

Reply:
Thanks, I understand now. I was confused by the wording and would suggest changing this to

```suggestion
      /**
       * When the user has uninitialized secrets, this will be reflected in
       * the notification bell. The user might just have deleted that secret
       * and we need to update the notification bell.
       */
```